client.go

```diff
@@ -101,7 +101,7 @@ type ProgressListener interface {
 // Describer of Functions' remote deployed aspect.
 type Describer interface {
 	// Describe the running state of the service as reported by the underlyng platform.
-	Describe(name string) (description Description, err error)
+	Describe(ctx context.Context, name string) (description Description, err error)
```

Comment:
I'm not sure what to say about bumping knative dependency - probably @rhuss or @markusthoemmes have more insight on that. But I think the changes in this file are probably unrelated, aren't they? I do like adding context in general, but does it need to be in this PR?

Reply:
I am updating the dep specifically to be able to use ctx.

Reply:
but I could do separate PR